participants/yolo.md

```diff
@@ -0,0 +1,52 @@
+
+## (2) Link to Project UI
+- [Yolo demo](https://yolo.coinsummer.io)
+- [Yolo front-end repository](https://github.com/CoinSummer/yolo)
+
+## (3) What does your application/UI do?
+Everyone can use Yolo dataset to train their own Machine-learning and AI models. Users who have images can also upload them to contribute the Yolo dataset together with us. There will be some token rewards (in the feature) for their contribution after these images been reviewed. The Yolo application works as following:
+![Yolo Network](https://raw.githubusercontent.com/CoinSummer/yolo/master/docs/yolo-network.png)
+
+## (4) If your project is using a curated dataset, which one are you using?
+We use [Open Images Dataset](https://storage.googleapis.com/openimages/web/index.html).
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+We are using a curated dataset.
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+We are planning to store at least 5TB data to the Filecoin network, maybe more than 10TB if everything goes well.
+
+## (7) How are you structuring the data?
+The images dataset are unstructured data with JPG or PNG format. However, we manage these dataset and our application data by database like MySQL.
+
+## (8) What pre-processing are you doing before ingesting the data?
+It is currently planned to package a series of images which have same type and category, and then store them to the Filecoin network by sending storage deals to miners. The size of storage deals will be 3GB to 5GB according to the network bandwidth when sending to miners.
+
+Every image which has packaged will be saved as a record in our database also. There is a `storageDealId` field to record that each image is contained in which deal. We get the deal status by subscribing Powergate's JSON RPC API. After one storage deal has successfully be sealed on the Filecoin sector, we will send a retrieve deal, and then store all the images in the deal to IPFS network as cache. The Yolo application will fetch images and expose them to users from IPFS network.
+
+There is some uncertainty about the API of Powergate, we have not yet completed the integration of Powergate's api, perhaps something will change our plan.
+
+## (9)  What tech stack will you use to store the data?
+We are planning to use Powergate and lotus.
+
+## (10) How will you retrieve the data?
+I think our application only need to retrieve data from Filecoin once. After the storage deal has successfully sealed by miner, we will send a retrievel deal, and then store all the data in the deal to IPFS. IPFS is a intermediate caching layer for us to retrieve the necessary data and expose it to users.
+
```

Comment:
Will retrieval be on a per image basis or on a set of images?

Reply:
Will retrieval be on a set of images. We are planning to package a set of images for about 3GB to 5GB, and store them as storage deals to Filecoin, and then retrieve these deals and un-package them as single image, each image will be stored to IPFS as cache for expose it to users. I don't clearly does Powergate's API support this case, please let us try.